Doc/glossary.rst

```diff
@@ -811,9 +811,10 @@ Glossary
       processed.
 
    loader
-      An object that loads a module. It must define a method named
-      :meth:`load_module`. A loader is typically returned by a
-      :term:`finder`. See also:
+      An object that loads a module. It must define the
+      :meth:`!exec_module` and :meth:`!create_module` methods
+      to implement the :class:`~importlib.abc.Loader` interface.
+      A loader is typically returned by a :term:`finder`. See also:
```

Comment:
@nikalinov note you can use the "Commit suggestion" button in the GH interface to apply a reviwer's suggestions. This keeps newlines to reduce the diff for any future changes.
```suggestion
      An object that loads a module.
      It must define the :meth:`!exec_module` and :meth:`!create_module` methods
      to implement the :class:`~importlib.abc.Loader` interface.
      A loader is typically returned by a :term:`finder`.
      See also:
```